contents/docker/Docker.md

```diff
@@ -0,0 +1,165 @@
+<frontmatter>
+  title: DevOps
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Docker
+
+Authors: [Rahul Rajesh](https://github.com/rrtheonlyone)
+
+In today’s age, web applications are becoming more complex. Web applications of recent times are starting to move from a monolithic architecture to one that is based off of microservices. 
+
+<box type="info">
+A Microservices architecture is when you split your application into a set of smaller, interconnected services that each have a specific role (as opposed to one big application). Read up more about it <a href="https://articles.microservices.com/monolithic-vs-microservices-architecture-5c4848858f59">here.</a>
+</box>
+
+With the increasing complexity of these applications, deployment starts to become a concern. Instead of running one service, some applications will require you to run a few services at the same time. There also needs to be configurations setup for each of these services. Applications like Docker help to make deployment a seamless process. This article provides an introduction to Docker along with the advantages that it is able to provide.
```

Comment:
This could be a nice way to start a presentation about docker (i.e., PUNCH), but I'm not sure if that works for an article. Why bring in an entirely new (and not strictly necessary) topic at the very start? If you remove it, the reader wont feel like something is missing.

Reply:
Yes you are right! I have removed this.